examples/tooling/scripts/sql-export.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+set -e
+
+# Get the lando logger
+. /helpers/log.sh
+
+# Set the module
+LANDO_MODULE="sqlexport"
+
+# Set generic things
+HOST=localhost
+SERVICE=$LANDO_SERVICE_NAME
+STDOUT=false
+
+# Get type-specific config
+if [[ ${POSTGRES_DB} != '' ]]; then
+  DATABASE=${POSTGRES_DB:-database}
+  PORT=${LANDO_DB_EXPORT_PORT:-5432}
+  USER=${LANDO_DB_EXPORT_USER:-postgres}
+else
+  DATABASE=${MYSQL_DATABASE:-database}
+  PORT=${LANDO_DB_EXPORT_PORT:-3306}
+  USER=${LANDO_DB_EXPORT_USER:-root}
+fi
+
+# Set the default filename
+FILE=${DATABASE}.`date +"%Y-%m-%d-%s"`.sql
+
+# PARSE THE ARGZZ
+# TODO: compress the mostly duplicate code below?
+while (( "$#" )); do
+  case "$1" in
+    # This doesn't do anything anymore
+    # we just keep it around for option validation
+    -h|--host|--host=*)
+      if [ "${1##--host=}" != "$1" ]; then
+        shift
+      else
+        shift 2
+      fi
+      ;;
+    --stdout)
+        STDOUT=true
+        shift
+      ;;
+    --)
+      shift
+      break
+      ;;
+    -*|--*=)
+      shift
+      ;;
+    *)
+      if [[ "$1" = /* ]]; then
+        FILE="${1//\\//}"
+      else
+        FILE="$(pwd)/${1//\\//}"
+      fi
+      shift
+      ;;
+  esac
+done
+
+# Get type-specific dump cpmmand
```

Comment:
Typo here: `cpmmand`

Reply:
Ah! That typo was copied over in the script in lando/core, it should be fixed there.

Reply:
I generally run https://github.com/codespell-project/codespell as part of my update process as I'm sure @xurizaemon is well aware. :smiley: